components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/actions/ExtendWorkspaceTimeoutAction.kt

```diff
@@ -31,10 +31,10 @@ class ExtendWorkspaceTimeoutAction : AnAction() {
                     thisLogger().error("gitpod: failed to extend workspace timeout", e)
                 } else {
                     if (result.resetTimeoutOnWorkspaces.isNotEmpty()) {
-                        message = "Workspace timeout has been extended to three hours. This reset the workspace timeout for other workspaces."
+                        message = "Workspace timeout is extended to 180 minutes. This reset the workspace timeout for other workspaces."
```

Comment:
> This reset the workspace timeout for other workspaces.

Is this still true? I assume workspace timeouts do not still apply just to one workspace still, do they? 

This will need documenting in [the timeouts documentation update](https://github.com/gitpod-io/website/pull/3326/files).

Reply:
[Internal discussion](https://gitpod.slack.com/archives/C02CBKZEVS8/p1675087163143529) 🧵

Reply:
I will keep this wording change as it is. And regarding the Reset workspace timeout message. We can followup on this issue: https://github.com/gitpod-io/gitpod/issues/16107